pinot-common/src/main/java/org/apache/pinot/common/messages/SegmentReloadMessage.java

```diff
@@ -34,19 +35,25 @@ public class SegmentReloadMessage extends Message {
   public static final String RELOAD_SEGMENT_MSG_SUB_TYPE = "RELOAD_SEGMENT";
 
   private static final String FORCE_DOWNLOAD_KEY = "forceDownload";
+  private static final String SEGMENT_LIST_KEY = "segmentList";
```

Comment:
(minor) Seems we are using `SEGMENT_NAMES` key in `ForceCommitMessage`, consider using the same key (but store as list field because that seems preferred)